Sources/Stats/mc_stats.hpp

```diff
@@ -27,8 +27,8 @@ struct Stats {
   double error_of_mean;
   /// Average of in-chain variances of the observable over all Markov Chains.
   double variance;
-  /// Autocorrelation
-  double correlation;
+  /// Autocorrelation time
+  int correlation;
```

Comment:
I suggest we stick to using a `double` here for the sake of having a NaN value. I don't have anything against rounding it though.